spec/models/follow_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+require 'rails_helper'
+
+describe Follow do 
+  it "is invalid without a target_id" do
+    follow = Follow.new
+    follow.should_not be_valid
+  end
+
+  it "is invalid without a user_id" do
+    follow = Follow.new
+    follow.should_not be_valid
+  end
+end
```

Comment:
This seems incomplete, you are actually not testing anything here.  You need to create a new object without a target_id and without a user_id.